### DISCUSSION TITLE
broken on iOS 14.5

### DISCUSSION THREAD

**Discussion by YuncongHan** [utmapp/UTM#2482] [UTM Help (iOS)] [answered]
**Describe the problem**
In ios14.5, unable to enter the software

**Configuration (required)**
* UTM version: v2.0.27
*Operating system version: ios14.5
*Device model: ipad pro2020
* Whether to jailbreak (use the jailbreak name)： No
*How did you install UTM? Via altstore

**Comment by JustinZhou88** [comment]
me neither.

**Comment by YuncongHan** [comment]
I think the factor is that iOS 14.5 isn't fully supported yet, and I'm still waiting to hear from others.

**Comment by JustinZhou88** [comment]
> I think the factor is that iOS 14.5 isn't fully supported yet, and I'm still waiting to hear from others.

I can enter the software, but I can’t launch it.

**Comment by YuncongHan** [comment]
I can't enter the software，

> > I think the factor is that iOS 14.5 isn't fully supported yet, and I'm still waiting to hear from others.
> 
> 
> 
> I can enter the software, but I can’t launch it.

**Comment by JustinZhou88** [comment]
Who can teach me how to “tethered to Xcode”, it’s difficult for me to understand. My god!

**Comment by ghost** [comment]
> Who can teach me how to “tethered to Xcode”, it’s difficult for me to understand. My god!

https://github.com/utmapp/UTM/blob/master/Documentation/TetheredLaunch.md

**Comment by ty-yqs** [comment, marked as answer]
It seemed that JIT broken on iOS 14.5

**Reply by osy** [reply]
See the pinned topic #2271
